pkg/controller/controller_clusterservicebroker.go

```diff
@@ -36,31 +36,31 @@ import (
 // the Message strings have a terminating period and space so they can
 // be easily combined with a follow on specific message.
 const (
-	errorFetchingCatalogReason  string = "ErrorFetchingCatalog"
-	errorFetchingCatalogMessage string = "Error fetching catalog. "
-	errorSyncingCatalogReason   string = "ErrorSyncingCatalog"
-	errorSyncingCatalogMessage  string = "Error syncing catalog from ServiceBroker. "
-
-	errorListingClusterServiceClassesReason  string = "ErrorListingServiceClasses"
-	errorListingClusterServiceClassesMessage string = "Error listing service classes."
-	errorListingClusterServicePlansReason    string = "ErrorListingServicePlans"
-	errorListingClusterServicePlansMessage   string = "Error listing service plans."
-	errorDeletingClusterServiceClassReason   string = "ErrorDeletingServiceClass"
-	errorDeletingClusterServiceClassMessage  string = "Error deleting service class."
-	errorDeletingClusterServicePlanReason    string = "ErrorDeletingServicePlan"
-	errorDeletingClusterServicePlanMessage   string = "Error deleting service plan."
+	errorFetchingClusterCatalogReason  string = "ErrorFetchingClusterCatalog"
```

Comment:
I think the error should still be `ErrorFetchingCatalog`

Reply:
same comment for the other errors

Reply:
Same - I don't think these constants should change.